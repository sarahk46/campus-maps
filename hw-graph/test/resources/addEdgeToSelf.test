# Tests that a graph along with a node and an edge to itself can be created.

# Create a graph
CreateGraph graph1

# Add one node
AddNode graph1 n1

# Add one edge
AddEdge graph1 n1 n1 e1