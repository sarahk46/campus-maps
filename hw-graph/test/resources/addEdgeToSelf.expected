# Tests that a graph along with a node and an edge to itself can be created.

# Create a graph
created graph graph1

# Add one node
added node n1 to graph1

# Add one edge
added edge e1 from n1 to n1 in graph1